configs/application/presentationComponents.json

```diff
@@ -546,16 +545,18 @@
 			"component": {
 				"category": "system",
 				"spawnOnStartup": true,
-				"spawnOnAllMonitors": false,
-				"canMinimize": false,
-				"canMaximize": false
+				"spawnOnAllMonitors": false
 			},
 			"foreign": {
 				"services": {
-					"dockingService": {
-						"isArrangable": false,
-						"ignoreSnappingRequests": true,
-						"ignoreTilingAndTabbingRequests": true
+					"windowService": {
```

Comment:
These can be deleted as well since they are all default options. You showed a sample those options in the welcome component and users can also find the available options in our documentation, so it's probably not necessary to put them here again.